lib/node/runner/api/transaction_post.go

```diff
@@ -1,15 +1,14 @@
 package api
 
 import (
-	"boscoin.io/sebak/lib/errors"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"boscoin.io/sebak/lib/common"
 	"boscoin.io/sebak/lib/network/httputils"
 	"boscoin.io/sebak/lib/node/runner/api/resource"
 	"boscoin.io/sebak/lib/transaction"
-	"bufio"
-	"bytes"
-	"encoding/json"
-	"io"
-	"net/http"
 )
 
 type TeeReadCloser struct {
```

Comment:
`TeeReadClose` is needed?

Reply:
No, we don’t and also InterceptXXX too. I just left these for future :)